src/Workspaces/Core/Portable/Rename/Renamer.RenameSymbolDocumentAction.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using System.Linq;
+using System;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Rename
+{
+    public static partial class Renamer
+    {
+        /// <summary>
+        /// Action that will rename a type to match the current document name
```

Comment:
```suggestion
        /// Action that will rename a type to match the current document name.
```

Reply:
i would flesh out this information a bunch more.  For example, it might be good to explain how we pick the symbol to rename and if there are any caveats there (i.e. does this work if there are multiple tyeps in hte file?  if the type to be renamed is not the first type in the file?  etc.).

Reply:
Applying the suggested change, but leaving this open for me to revisit in the morning and flesh out more.